KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,47 @@
+# Kubeflow Outreach Committee (KOC)
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help onboard new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
```

Comment:
Collaborate with other committees and working groups should be a core principal and a requirement for all KOC actions.

That is, any outreach effort should involve the leads of the relevant working groups. For example, if an outreach event is for Kubeflow Pipelines, it would need the blessing of the Pipelines WG.

More specifically, all the other charter elements should effectively say "in collaboration with the relevant working groups".

Reply:
I think this will end up true in practice but we don't need to be over the top with this.

Reply:
Yes, I also want to avoid death by overregulation  ;-)